src/quicktype-core/language/Python.ts

```diff
@@ -274,9 +274,9 @@ export class PythonRenderer extends ConvenienceRenderer {
 
     protected emitDescriptionBlock(lines: string[]): void {
         if (lines.length === 1) {
-            this.emitLine('"""', lines[0], '"""');
+            this.emitLine("\"\"\"", lines[0], "\"\"\"");
```

Comment:
Please remove these from the PR.  We use "prettier" for formatting, and it'll will format it back to the original next time we edit the file.